setup.py

```diff
@@ -31,8 +31,9 @@
         'cdo',
         'cf_units',
         'cython',
-        # 'scitools-iris',  # Only iris 2 is on PyPI
+        'scitools-iris',
         'matplotlib<3',
+        'nc-time-axis',
```

Comment:
Why is this required? I cannot find a module importing this?

Reply:
`iris.plot` needs it when dealing with Gregorian calendars

Reply:
see the old issue https://github.com/ESMValGroup/ESMValTool/issues/396

Reply:
Can you add a comment saying that? And also add the package to meta.yml? Else it will still be broken if you install esmvaltool using the conda package.